test/lib/index.ts

```diff
@@ -121,6 +121,18 @@ test.serial("ui5lint API: Ignore config file for com.ui5.troublesome.app", async
 	t.snapshot(preprocessLintResultsForSnapshot(res));
 });
 
+test.serial("ui5lint API: Use defaults", async (t) => {
+	const projectPath = path.join(fixturesProjectsPath, "com.ui5.troublesome.app");
+	const {ui5lint, sinon} = t.context;
+	// Stub process.cwd(), so we have some app to test
+	sinon.stub(process, "cwd").returns(projectPath);
+
+	const res = await ui5lint({});
```

Comment:
Please also add another test case where no arguments are provided, as this is what previously failed.

Reply:
Thanks!
Sorry, I have forgotten to remove that